### ISSUE TITLE
Avoid dependency on Lake in CurrentModule

### ISSUE THREAD

**Issue by jtristan** [leanprover-community/import-graph#20]
The [use of the definition](https://github.com/leanprover-community/import-graph/blob/68b518c9b352fbee16e6d632adcb7a6d0760e2b7/ImportGraph/CurrentModule.lean#L15) `Lake.Manifest.load?` creates a dependency on `Lake`. For work that needs todo reverse FII with Mathlib, it is an issue because Lean doesn't create a dynamic library for the Lake library. It is possible to work around it, but it is less than ideal. Perhaps the use of this definition [could be avoided](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/Calling.20Lean4.20code.20from.20Python/near/452013224)?

**Comment by joneugster** [comment]
definitely, the current definition is a hack as good as any other. There is no direct reason why it needs to go via the lake manifest.

However, I think it would be desirable if it was replaced by some logic that gets the correct module for reasonably set up projects.

**Comment by joneugster** [comment]
I think this has been resolved by including Lake shared library in the Lean4 distribution since `v4.13.0`: [see comment](https://github.com/leanprover-community/batteries/pull/811#issuecomment-2415899180)

Therefore I suggest to close this issue.
